### ISSUE TITLE
Eigenscape Park 6 and 8 recordings are not really B-format

### ISSUE THREAD

**Issue by iranroman** [soundata/soundata#98]
As indicated in the [micarraylib](https://github.com/micarraylib/micarraylib) issue [#8](https://github.com/micarraylib/micarraylib/issues/8) and acknowledged by @marc1701 , the author of the dataset.

Eigenscape Park 6 and 8 recordings are not really B-format. 

This issue has been addressed this week by @marc1701, with a new version of the dataset on the [official Zenodo page](https://zenodo.org/record/2628463#.YkW7OS1h1pR).

Action items:

- [x] download Park 6 and Park 8 files from the latest version of the Eigenscape dataset and check that they only have 25 channels, as every other file in `Park.zip`.
- [x] update index and loader of Eigenscape dataset in soundata. 
- [x] check with the author if the same fixes can be done for the `Eigenscape_Raw` version of the dataset, which also has a mismatch in the number of channels for the Park6 and Park8 files.

**Comment by iranroman** [comment]
@sakshamsingh1, can you please carry out action items 1 and 2? I'll work on action item 3.

**Comment by sakshamsingh1** [comment]
Hi @iranroman, I was able to verify item 1 i.e. Park.6 and Park.8 have 25 channels alike other files in Park.zip.
Code :
```
import librosa
import os
base_path = 'park_data/'
print('Num of channels')
for file in os.listdir(base_path):
    path = base_path + file
    y,r = librosa.load(path,sr=None,mono=False)
    print(f' {file} : {y.shape[0]}')
```

Output :
```
Num of channels
 Park.8.wav : 25
 Park.2.wav : 25
 Park.1.wav : 25
 Park.5.wav : 25
 Park.4.wav : 25
 Park.3.wav : 25
 Park.7.wav : 25
 Park.6.wav : 25
```

**Comment by iranroman** [comment]
Hello all. @marc1701 (EigenScape author) has confirmed that the Park6 and Park8 files were accidentally swapped  between raw and B formats. So I'll go ahead and swap the urls for these files in the loaders for the `Park` recordings between these loaders. That should get us a quick and dirty "engineered" solution. For more details about why this is the best solution, see [`micarraylib` issue #8](https://github.com/micarraylib/micarraylib/issues/8#issuecomment-1095422983)

**Comment by harshpalan** [comment]
Closed with #102
